packages/ui/src/components/cell/cell.native.tsx

```diff
@@ -1,108 +1,84 @@
-import { Switch } from 'react-native';
+import { ReactNode } from 'react';
 
 import { ResponsiveValue, useTheme } from '@shopify/restyle';
 
-import { Box, ChevronRightIcon, IconProps, Text, Theme, TouchableOpacity } from '../../../native';
-import { TouchableOpacityProps } from '../button/touchable-opacity.native';
+import {
+  Avatar,
+  ChevronRightIcon,
+  Flag,
+  IconProps,
+  ItemLayout,
+  RadioButton,
+  Theme,
+  Switch as UISwitch,
+} from '../../../native';
+import { Pressable, PressableProps } from '../button/pressable.native';
+import { RadioButtonProps } from '../radio-button/radio-button.native';
+import { SwitchProps } from '../switch/switch.native';
 
 type RegularCellVariant = 'active' | 'inactive' | 'critical';
-type SwitchCellVariant = 'switch';
-type CellVariant = RegularCellVariant | SwitchCellVariant;
 
 function getIconColor(
-  variant: CellVariant
+  variant: RegularCellVariant
```

Comment:
What does `Regular...` mean here? I think there are some naming inconsistencies that make the code hard to follow. Here, the three type options don't all seem to make sense together? `active` and `inactive` make sense, but what does `critical` mean then in relation to those two states? Can the cell be both `active` and `critical`?